astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -87,20 +84,20 @@ class galactocentric_frame_defaults(ScienceState):
 
     """
 
-    # the default is to use the original definition of this frame
-    # TODO: change this to 'latest' in v4.1?
-    _value = 'pre-v4.0'
+    _latest_value = 'v4.0'
     _references = None
+    _value = None
 
-    @staticmethod
-    def get_solar_params_from_string(arg):
+    @classmethod
```

Comment:
Why is it a class method? Doesn't seem to be returning a new instance of that class.

Reply:
Right - I wanted to define the default value at the class level and use `cls._latest_value` internally, but that was just for aesthetics - I could move the `latest_value` definition into the method and change it back to a staticmethod...

Reply:
Can it just be a normal method with `self`?

Reply:
No, not with the way `ScienceState`'s work (as far as I remember, they use the *class* to store state)

Reply:
But an instance can access the class variable. Am I missing something?

```python
class A:
    latest = '4.0'

    def get_stuff(self):
        return self.latest
```
```python
>>> aa = A()
>>> aa.get_stuff()
'4.0'
```

Reply:
Yes - [read how `ScienceState` is implemented](https://github.com/astropy/astropy/blob/cfb1c2faff72076c3931e51c21fb6950b90c4295/astropy/utils/state.py) and try to create an instance

Reply:
Oh... Well, if `ScienceState` says so... Thanks for the clarification!